docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -48,60 +59,87 @@ npx create-strapi-app@latest my-project --quickstart
 
-The admin panel of Strapi runs at [http://localhost:1337/admin](http://localhost:1337/admin). This is where you will spend most of your time creating and updating content.
+First we will build a data structure for your content. This can only be done while in development mode, which is the default mode for projects that are created locally.
 
 :::tip TIP
 If the server is not already running, in your terminal, `cd` into the `my-project` folder and run `npm run develop` (or `yarn develop`) to launch it.
 :::
 
-### Step 1: Create collection types with the Content-type Builder
+The Content-Type Builder helps you create your data structure. When creating an empty project with Strapi, this is where to get the party started!
+
+<details style={detailsStyle}>
 
-The Content-type Builder plugin helps you create your data structure. When creating an empty project with Strapi, this is where to get the party started!
+<summary style={summaryStyle}>Step 1: Create a "Restaurant" collection type</summary>
 
-#### Create a "Restaurant" collection type
+### Step 1: Create a "Restaurant" collection type
 
 Your restaurants directory will eventually include many restaurants, so we need to create a "Restaurant" collection type. Then we can describe the fields to display when adding a new restaurant entry:
 
-1. Click on the **Create your first Content type** button.<br />If it's not showing up, go to Plugins ![Content-type Builder icon](/img/assets/quick-start-guide/icons/content_types_builder.svg) [Content-type Builder](http://localhost:1337/admin/plugins/content-type-builder) in the main navigation.
+1. Click on the **Create your first Content type** button.<br />If it's not showing up, go to ![Content-type Builder icon](/img/assets/quick-start-guide/icons/content_types_builder.svg) [Content-type Builder](http://localhost:1337/admin/plugins/content-type-builder) in the main navigation.
 2. Click on **Create new collection type**.
 3. Type `Restaurant` for the _Display name_, and click **Continue**.  
 4. Click the Text field.
 5. Type `Name` in the _Name_ field.
 6. Switch to the _Advanced Settings_ tab, and check the **Required field** and the **Unique field** settings.
 7. Click on **Add another field**.
-8. Choose the Rich text field.
+8. Choose the Rich text (Blocks) field in the list.
 9. Type `Description` under the _Name_ field, then click **Finish**.
 10. Finally, click **Save** and wait for Strapi to restart.
 
+<ThemedImage
+alt="GIF: Create Restaurant collection type in Content-type Builder"
+sources={{
+    light: '/img/assets/quick-start-guide/qsg-handson-restaurant_2.gif',
+    dark: '/img/assets/quick-start-guide/qsg-handson-restaurant_2_DARK.gif',
+}}
+/>
 ![GIF: Create Restaurant collection type in Content-type Builder](/img/assets/quick-start-guide/qsg-handson-restaurant_2.gif)
```

Comment:
I think you forgot to delete this line: we can see it written below the gif:
<img width="707" alt="Screenshot 2024-01-25 at 11 00 26" src="https://github.com/strapi/documentation/assets/19183360/8af2e85a-52e5-4373-b7c3-1df9585fe5c1">